Makefile

```diff
@@ -161,14 +161,14 @@ IGNORE_CLEANUP_LIBS := CMSIS FreeRTOS STM32F0xx_StdPeriph_Driver unity FatFs
 # This uses regex
 IGNORE_PY_FILES := ./lint.py ./libraries/unity.*
 # Find all python files excluding library files in project env (./venv)
-FIND_PY_FILES:= $(shell printf "! -regex %s " $(IGNORE_PY_FILES) | xargs find -path ./venv -prune -o -name '*.py')
+FIND_PY_FILES:= $(shell printf "! -regex %s " $(IGNORE_PY_FILES) | xargs find . \( -path ./.venv -o -path ./libraries -o -path ./lint.py \) -prune -o -name '*.py' -print)
```

Comment:
The point of `IGNORE_PY_FILES` is to filter out ./libraries/unity/* and ./lint.py, is there a reason you added those to the -prune too? At least you should use `IGNORE_PY_FILES` somehow (perhaps with a gnu make function) instead of duplicating.